mod/01_core/set/all/fetch.rb

```diff
@@ -54,8 +54,11 @@ def fetch mark, opts={}
     if card.new_card?
       if opts[:new]
         return card.renew(opts) if !clean_cache_opts? opts
+      elsif opts[:skip_virtual]
```

Comment:
Isn't the second case one where :skip_virtual is implied?  Or is this a case that this branch is fixing?  known? is true for virtual cards, right?  Now we will include the modules anyway for virtual cards.

Yeah, these do seem like distinct cases.

Do we often do known?(name) && fetch(name) (or similar).  Seems like we should be able to get the card without another fetch when known? returns truthy.  It could just return card or nil, maybe a version without the question mark?